modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -252,7 +252,7 @@ def valid_526_response
 
         def format_526_errors(errors)
           errors.map do |error|
-            { status: 422, detail: "#{error['key']} #{error['detail']}", source: error['key'] }
+            { status: 422, detail: "#{error[:key]} #{error[:detail]}", source: error[:key] }
```

Comment:
@stiehlrod  Is this going to work for previously saved error message?  (Is that a concern?)